sceptre/cli/list.py

```diff
@@ -78,11 +93,18 @@ def list_change_sets(ctx, path):
     List change sets for stack.
 
     """
-    stack, _ = get_stack_or_stack_group(ctx, path)
+    context = SceptreContext(
+                command_path=path,
+                project_path=ctx.obj.get("project_path", None),
```

Comment:
Not a big deal, by deafult `..get(...)` will always result in `None` if it can't be found, so the `None` specified here and elsewhere is redundant.

Reply:
resolved.